.github/workflows/wait_for_required_workflows.yml

```
@@ -81,7 +87,7 @@ jobs:
               const runs = checkRuns.map(({ name, conclusion }) => ({ name, conclusion }))
               console.log(`Got the following check runs: ${JSON.stringify(runs)}`)
               const matchingRuns = runs.filter(({ name }) => actions.includes(name))
-              const failedRuns = matchingRuns.filter(({ conclusion }) => conclusion !== 'success')
+              const failedRuns = matchingRuns.filter(({ name, conclusion }) => !(conclusion == 'success' || (can_skip.includes(name) && conclusion == 'skipped')))
```

```suggestion
              const failedRuns = matchingRuns.filter(({ name, conclusion }) => !(conclusion === 'success' || (canSkip.includes(name) && conclusion === 'skipped')))
```

`==` does type coercion in JS which we don't want